src/test/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionServiceTest.java

```diff
@@ -112,6 +113,9 @@
     @Mock
     private IReferenceNumberGeneratorService referenceNumberGeneratorService;
 
+    @Mock
+    private Gaz2RequestedValidator gaz2RequestedValidator;
+
```

Comment:
needs a test to check that gaz2RequestedValidator.validate is being called during the createObjection and isCompanyEligible methods.
Also need to check that the correct status gets set during those methods if gaz2RequestedValidator throws validation exception.